apps/browser/lib/hooks/useEvoService.ts

```diff
@@ -47,7 +47,7 @@ export const useEvoService = (
   // State
   const [isStarting, setIsStarting] = useState<boolean>(false);
   const [isRunning, setIsRunning] = useState<boolean>(false);
-  const [chatLog, setChatLog] = useState<ChatLog[]>([]);
+  const [chatLog, setChatLogState] = useState<ChatLog[]>([]);
```

Comment:
Why the rename? Also, should we make `chatLog` be `chatLogState` too?

Reply:
It's just so that it doesn't conflict with our custom `setChatLog` function below.